config/site_settings.yml

```diff
@@ -1771,6 +1771,10 @@ security:
     default: false
     client: true
     hidden: true
+  suppress_secured_categories_from_admin:
+    default: false
+    client: true
```

Comment:
Sorry one more note, does this need to be shipped to the client?

Reply:
ohh cut and paste fail ... no not client side thing will fix